src/v2/authentication.rs

```diff
@@ -95,15 +107,15 @@ impl Token {
 		self.claims.exp = self.expiry.unix_timestamp();
 
 		let jwt = encode(&self.header, &self.claims, &self.key)?;
+		let scope = MIN_SCOPE.to_owned() + &self.scope.clone().unwrap_or_default().join(" ");
```

Comment:
not as pretty but a bit more robust IMO
```suggestion
		let scope =
			[[MIN_SCOPE.to_owned()].as_ref(), self.scope.clone().unwrap_or_default().as_slice()]
				.concat()
				.join(" ");
```